linux/kinto.py

```diff
@@ -1070,7 +1070,7 @@ def _is_Enter_F2():
 
 keymap("Modal dialog fix - GNOME Settings child windows",{
     C("RC-W"):                  C("Alt-F4"),
-}, when = lambda ctx: ctx.wm_class == "^Gnome-control-center$" and not ctx.wm_name == "^Settings$")
+}, when = lambda ctx: ctx.wm_class == "Gnome-control-center" and not ctx.wm_name == "Settings")
```

Comment:
Correct.  `==` is not a regex check. :)

Reply:
Got a bit overzealous adding those to everything to match the current config. Forgot that was one of the reasons I made `isWindow()`. Which I need to make smarter to deal with the “not” situation.